tests/unit/flask/constants.py

```diff
@@ -8,7 +8,7 @@
         "flask": {
             "override": "replace",
             "startup": "enabled",
-            "command": f"/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app",
+            "command": f"/bin/python3 -m gunicorn -c /flask/gunicorn.conf.py app:app -k sync",
```

Comment:
Are there any tests without the `-k sync` for backwards compatibility please?

Reply:
The latest paas-charm instance automatically adds `-k sync` to command as can be seen [here](https://github.com/canonical/paas-charm/blob/88d898f105c40294c56e9b8d198bf4a9d4df108d/src/paas_charm/_gunicorn/wsgi_app.py#L51). How can I test it for backwards compatibility with this?
Also  thanks to you I just realised I need to put an empty space before adding the `-k sync` for backwards compatibility. Thanks!

Reply:
Is this what is expected to be added?